core/src/templates/components/hero/hero.twig

```diff
@@ -69,7 +69,6 @@
       "card_cta_link": hero_cta_link,
       "card_cta_attributes": hero_cta_attributes,
       "card_cta_label": hero_cta_label,
-      "card_button_link": hero_button_link,
```

Comment:
You can add the deprecation notice but this change will have to wait until `v6`.